CONTRIBUTING.md

```diff
@@ -173,3 +173,41 @@ Before merging a PR make sure:
 
 Merge the PR by using "Squash and merge" option on Github. Avoid creating merge commits.
 After the merge make sure referenced issues were closed.
+
+### Developing guidelines
+
+#### Building OCI Images for multiple arch (linux/arm64, linux/amd64)
```

Comment:
This section needs some better formatting and grammar check (@objectiser?), but I'm fine in having this done by someone else or in a future PR.